charts/auth-layer-server/README.md

```diff
@@ -0,0 +1,245 @@
+
+Uses `smtp` host, port and optionally credentials (username and password) to send emails.
+Is also required to confiture `From` and recommended to configure `From Display Name` fields.
+
+*Once the email service is configured, is possible to test the email service by sending a test email from the `Email` tab to verify that the email service is working.*
+
+
+### Usage
+
+#### Create a new user
+
+Once you have your KeyCloak server running, you can create a new user by following these steps:
+
+1. Login to the KeyCloak admin console.
+2. Select the `HederaTheGraph` realm from the dropdown menu (if not selected already).
+3. Then navigate to `Users` and click on the `Add user` button.
+4. Fill in the user details and click on the `Save` button.
+5. After saving the user, navigate to the `Credentials` tab and set a password for the user (prefferably a temporary one, that forces the user to update it at next login).
+6. Assign Roles or Grpups to the user.
+7. Set `subgraph_access` attribute to the user, this will give the user access to these subgraphs. This is done by click the `Attributes` tab and adding a new attribute with the name `subgraph_access` and the value being the subgraph id. The value can be a CSV string with many subgraph names, ie: `subgraph1,subgraph2,subgraph3`.
+
+#### Groups and Roles
+
+**a. Subgraph Admin:**, has access to all the subgraphs admin endpoints. (`subgraph_create`, `subgraph_remove`, `subgraph_deploy`,`subgraph_pause` and `subgraph_resume`)
+**b. Subgraph Deployer:**, has access to: `subgraph_pause`, `subgraph_resume` and `subgraph_deploy` endpoints.
+**c. Subgraph Operator:**, has access to: `subgraph_pause` and 'subgraph_resume' endpoints.
+
+#### Get a Token from the Authentication Layer Server
+
+If using port-forwarding, you can get a token from the Authentication Layer Server by running the following command, you will need the following information:
+- `username` or `email` of the user
+- `user-password` of the user
+- `client-secret` of the `htg-auth-layer` client
+
+
+```bash
+curl --location 'http://localhost:8080/realms/HederaTheGraph/protocol/openid-connect/token' \
+--header 'Content-Type: application/x-www-form-urlencoded' \
+--data-urlencode 'grant_type=password' \
+--data-urlencode 'username=<username or email>' \
+--data-urlencode 'password=<user-password>' \
+--data-urlencode 'client_id=htg-auth-layer' \
+--data-urlencode 'client_secret=<client-secret>'
+```
+
+*However is recommended to expose the service using a FQDN since the token that is created using the host `localhost` is only valid for the `localhost` host when calling the instrospection endpoint. So port-forwarding is only recommended for testing purposes.*
+
+
+#### Verify a Token
+To verify a token, you can call the introspection endpoint of the Authentication Layer Server. The introspection endpoint is available at the following URL: `http://localhost:8080/auth/realms/HederaTheGraph/protocol/openid-connect/token/introspect`
```

Comment:
nit: this also would probably be better as a link

Reply:
This is not mean to be called by a browser, so the link does not makes too much sense.